apps/func-response-processor/src/index.ts

```
@@ -453,6 +462,102 @@ const serviceBusTrigger = async function (
+        redisMulti.hincrby(`${instanceKey}:results`, 'participants', 1)
+
+        if (participantData) {
+          const pointsPercentage = gradeQuestionCaseStudy({
+            response: response.assessment,
+            solutions: parsedSolutions,
+          })
+
+          pointsAwarded = computeAwardedPoints({
+            firstResponseReceivedAt,
+            responseTimestamp,
+            maxBonus: isNaN(parseInt(instanceInfo.maxBonusPoints, 10))
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)